### ISSUE TITLE
Incomplete installation instructions and broken dependencies

### ISSUE THREAD

**Issue by incrazyboyy** [TUMFTM/ScenarioArchitect#4]
As I was unable to install the requirements on Python 3.10, I resorted back to Python 3.7. Then, I had to install python3.7-dev in order to build the wheel for quadprog 0.1.7. Trying to run the GUI, I noticed that python3.7-tk is also necessary. But it still didn't open because of the following error:
`ImportError: /path/to/python3.7/site-packages/quadprog.cpython-37m-x86_64-linux-gnu.so: undefined symbol: _Z7qpgen2_PdS_PiS0_S_S_S_S_S_S0_S0_S0_S0_S0_S0_S_S0_`

I was able to solve this by bumping the trajectory-planning-helpers package version to 0.80, which requires quadprog 0.1.11. I cloned the [trajectory-planning-helpers repository](https://github.com/TUMFTM/trajectory_planning_helpers) and installed it manually, since the 0.80 version isn't available on pypi. quadprog 0.1.11 has pre-built wheels, making the python3.7-dev package obsolete.

So in summary, to get it working, I did:
1. Install python3.7 python3.7-tk
2. Clone and install the latest version of trajectory-planning-helpers manually
3. Change the version of trajectory-planning-helpers in requirements.txt to 0.80

My system is running **Ubuntu 22.04**.

I think the requirements and installation instructions need to be updated.
